frontend/sw360-portlet/src/main/webapp/html/utils/includes/usingProjectsTable.jspf

```diff
@@ -9,29 +9,55 @@
 --%>
 
 <core_rt:if test="${usingProjects != null  && usingProjects.size()>0}">
-    <table class="table info_table" title="Usage overview">
+    <table class="table info_table" style="margin-bottom: 0; border-bottom-width: 0">
         <thead>
         <tr>
-            <th colspan="3"> ${documentName} is used by the following projects</th>
+            <th colspan="3">${documentName} is used by the following projects</th>
         </tr>
-        <tr>
-            <th>Project name</th>
-            <th>Group</th>
-            <th>Responsible</th>
-        </tr>
-        </thead>
-        <core_rt:forEach var="project" items="${usingProjects}" varStatus="loop">
-            <tr>
-                <td>
-                    <sw360:DisplayProjectLink project="${project}"/>
-                </td>
-                <td>
-                    <sw360:out value="${project.businessUnit}"/>
-                </td>
-                <td>
-                    <sw360:DisplayUserEmail email="${project.projectResponsible}" bare="true"/>
-                </td>
-            </tr>
-        </core_rt:forEach>
     </table>
+
+    <div id="usingProjectsTableDiv">
+        <table id="usingProjectsTable" title="Usage overview" class="table info_table">
+        </table>
+    </div>
+
+    <script>
+        var usingProjectsTable;
+
+        function createUsingProjectsTable() {
+            var result = [];
+
+            <core_rt:forEach items="${usingProjects}" var="project">
+            result.push({
+                "DT_RowId": "${project.id}",
+                "id": '${project.id}',
+                "name": "<sw360:DisplayProjectLink project="${project}"/>",
+                "group": '<sw360:out value="${project.businessUnit}"/>',
+                "responsible": '<sw360:DisplayUserEmail email="${project.projectResponsible}" bare="true"/>'
+            });
+            </core_rt:forEach>
+
+            usingProjectsTable = $('#usingProjectsTable').DataTable({
+                "sPaginationType": "full_numbers",
+                "aaData": result,
+                "bFilter": false,
+                "bLengthChange": false,
+                search: {smart: false},
+                "aoColumns": [
+                    {title: "Project name", data: "name"},
+                    {title: "Group", data: "group"},
+                    {title: "Responsible", data: "responsible"}
+                ]
+            });
+
+            $('#usingProjectsTable_filter').hide();
```

Comment:
This is not necessary with proper datatables api usage. I have just recently removed all occurences of this kind of `.hide()`. See commit https://github.com/sw360/sw360portal/pull/476/commits/efbd3da7cc679f9df0944bd418d198414ca98e74 for examples.